### ISSUE TITLE
Strict connection checker demo should have info explaining what it does

### ISSUE THREAD

**Issue by BeksOmega** [rachel-fenichel/blockly-samples#1071]
**Category**

 - Examples

**Component**

strict-connection-checker

**Describe the bug**

The strict connection checker demo should have some text (like keyboard navigation) to indicate what it is doing. 

**To Reproduce**

1. Go to the strict connection checker demo
2. Observe how there is no info about what is being demo-ed.

**Additional context**

N/A
